docs/huggingface/tgi-0.9.3.md

```diff
@@ -0,0 +1,42 @@
+# HuggingFace Text-Generation-Inference (TGI) container Tuning guide
+
+The follow environment variables are the ones that you can set to use.
```

Comment:
Replace with: `Users can tune their TGI containers using the following environment variables:`